CMakeLists.txt

```diff
@@ -241,7 +247,9 @@ if(BML_OMP_OFFLOAD)
 
   add_definitions(-DUSE_OMP_OFFLOAD)
 
-  find_package(OpenMP 4.5 REQUIRED)
+  if(OpenMP_C_VERSION LESS 4.5)
```

Comment:
So we don't need find_package for OpenMP anymore?

Reply:
Line 160 should take care of it as BML_OPENMP is now required with BML_OMP_OFFLOAD